app/src/main/java/allsense/shopwithfriends/ItemDataSource.java

```diff
@@ -128,7 +128,8 @@ public void reportSale(final Item item, final User friend) {
             for (Item item : friendSales) {
                 // Check each item against user's interests
                 for (Interest interest : interests) {
-                    if (item.name().equals(interest.name()) && item.price() < interest.price()) {
+                    if (item.name().toLowerCase().equals(interest.name().toLowerCase())
+                            && item.price() < interest.price()) {
```

Comment:
This is good. Can you change the price comparison to <= (I meant to when I wrote the function)?